src/modules/tensor/cpu/kernel/flip.cpp

```diff
@@ -424,7 +424,7 @@ RppStatus flip_f32_f32_host_tensor(Rpp32f *srcPtr,
                     dstPtrTempG += vectorIncrementPerChannel;
                     dstPtrTempB += vectorIncrementPerChannel;
                 }
```

Comment:
what is this -2 corresponds to

Reply:
Updated magic number to const variable - It's the number of elements to skip